tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -123,6 +130,45 @@ describe('Eventing tests', function() {
     });
   }
 
+  function testStreamNotReCreated() {
+    context('stream check with JetStream backend', function() {
+      let wantStreamData = null;
+      let gotStreamData = null;
+      before('check if stream creation timestamp is available', async function() {
+        try {
+          const cm = await getConfigMap(streamDataConfigMapName);
+          wantStreamData = cm.data;
+        } catch (err) {
+          console.log('Skipping the stream recreation check due to missing configmap!');
+          this.skip();
+        }
+        if (skipStreamReCreationTest(wantStreamData)) {
+          console.log('Skipping the stream recreation check due to missing info!');
+          this.skip();
+        }
+      });
+
+      it('Get the current stream creation timestamp', async function() {
+        const vs = await createAPIRuleForService(eventingNatsApiRuleAName,
+            kymaSystem,
+            eventingNatsSvcName,
+            8222);
+        const vsHost = vs.spec.hosts[0];
+        gotStreamData = await getJetStreamStreamData(vsHost);
+      });
+
+      it('Compare the stream creation timestamp', async function() {
+        assert.equal(gotStreamData.streamName, wantStreamData.streamName);
+        assert.equal(gotStreamData.streamCreationTime, wantStreamData.streamCreationTime);
+      });
+
+
+      it('Delete the APIRule created', async function() {
```

Comment:
```suggestion
      it('Delete the created APIRule', async function() {
```
Or do I get this wrong?